yt_dlp/extractor/servus.py

```diff
@@ -15,130 +12,97 @@ class ServusIE(InfoExtractor):
                         (?:www\.)?
                         (?:
                             servus\.com/(?:(?:at|de)/p/[^/]+|tv/videos)|
-                            (?:servustv|pm-wissen)\.com/videos
+                            (?:servustv|pm-wissen)\.com/(?:[^/]+/)?v(?:ideos)?
                         )
-                        /(?P<id>[aA]{2}-\w+|\d+-\d+)
+                        /(?P<id>((AA|aa)-[A-Za-z0-9]{13})|([0-9]+-[0-9]+)|([A-Za-z]{2}[0-9A-Za-z]{18}))
```

Comment:
> @[pukkandan](https://github.com/pukkandan) pukkandan [on Jul 10, 2022](https://github.com/yt-dlp/yt-dlp/pull/4240#discussion_r917382041)
This change wasn't necessary imo, but you dont have to revert it either


> @[pukkandan](https://github.com/pukkandan) pukkandan [on Jul 10, 2022](https://github.com/yt-dlp/yt-dlp/pull/4240#discussion_r917382067)
Make all unused groups non-capturing

Reply:
Fixed in 0735abc4